src/features/helios/components/exoticShop/component/ExoticSeeds.tsx

```diff
@@ -37,59 +29,36 @@ export const ExoticSeeds: React.FC<Props> = ({ onClose }) => {
   const collectibles = state.collectibles;
 
   const price = selected.sfl;
-  const buy = () => {
-    gameService.send("bean.bought", {
-      bean: selected.name,
-    });
-
-    shortcutItem(selected.name);
-  };
 
   const lessFunds = () => {
     if (!price) return false;
 
     return state.balance.lessThan(price.toString());
   };
 
-  const lessIngredients = () => {
-    return getKeys(selected.ingredients).some((ingredientName) => {
-      const inventoryAmount =
-        inventory[ingredientName]?.toDecimalPlaces(1) || new Decimal(0);
-      const requiredAmount =
-        selected.ingredients[ingredientName]?.toDecimalPlaces(1) ||
-        new Decimal(0);
-      return new Decimal(inventoryAmount).lessThan(requiredAmount);
+  const lessIngredients = () =>
```

Comment:
Implicit returns really only make sense when it is a simple one liner. I think to be consistent, we should put this inside curly brackets and return.
```suggestion
  const lessIngredients = () => {
      return getKeys(selected.ingredients).some((name) =>
          selected.ingredients[name]?.greaterThan(inventory[name] || 0)
    ); 
  }
```